basics/docs-style.md

```diff
@@ -0,0 +1,201 @@
+
+### Subject
+
+In Bazel documentation, the audience should primarily be users—the people using
+Bazel to build their software.
+
+-  Address your reader as "you". (If for some reason you can't use "you",
+   use gender-neutral language, such as they.)
+   -  **YES:** To build Java code using Bazel, you must install a JDK.
+   -  MAYBE: For users to build Java code with Bazel, they must install a JDK.
+   -  **NO:** For a user to build Java code with Bazel, he or she must install
+      a JDK.
+
+-  If your audience is NOT general Bazel users, define the audience at the
+   beginning of the page or in the section. Other audiences can include
+   maintainers, contributors, migrators, or other roles.
+-  Avoid "we". In user docs, there is no author; just tell people what's
+   possible.
+   -  **YES:** As Bazel evolves, you should update your code base to maintain
+      compatibility.
+   -  **NO:** Bazel is evolving, and we will make changes to Bazel that at
+      times will be incompatible and require some changes from Bazel users.
+
+### Temporal
+
+Where possible, avoid terms that orient things in time, such as referencing
+specific dates (Q2 2020) or saying "now", "currently", or "soon."  These go
+stale quickly and could be incorrect if it's a future projection. Instead,
+specify a version level instead, such as "Bazel X.x and higher supports
+<feature>" or a GitHub issue link.
+
+-  **YES:** Bazel 0.10.0 or later supports remote caching.
+-  **NO:** Bazel will soon support remote caching, likely in October 2017.
+
+### Tense
+
+-  Use present tense. Avoid past or future tense unless absolutely necessary
+   for clarity.
+   -  **YES:** Bazel issues an error when it finds dependencies that don't
+      conform to this rule.
+   -  **NO:**  If Bazel finds a dependency that does not conform to this rule,
+      Bazel will issue an error.
+
+-  Where possible, use active voice (where a subject acts upon an object) not
+   passive voice (where an object is acted upon by a subject). Generally,
+   active voice makes sentences clearer because it shows who is responsible. If
+   using active voice detracts from clarity, use passive voice.
+   -  **YES:** Bazel initiates X and uses the output to build Y.
+   -  **NO:** X is initiated by Bazel and then afterward Y will be built with
+      the output.
+
+### Tone
+
+Write with a business friendly tone.
+
+-  Avoid colloquial language. It's harder to translate phrases that are
+   specific to English.
+   -  **YES:** Good rulesets
+   -  **NO:** So what is a good ruleset?
+
+-  Avoid overly formal language. Write as though you're explaining the
+   concept to someone who is curious about tech, but doesn't know the details.
+
+## Formatting
+
+### File type
+
+Where possible, use Markdown instead of HTML in your files. Follow the
+[GitHub Markdown Syntax Guide](https://guides.github.com/features/mastering-markdown/#syntax)
+for recommended Markdown style.
```

Comment:
Does Bazel have any additional style constraints we should talk about like line length limitations?
If not, or maybe regardless, does it makes sense to add any?

Maybe 80 columns for prose to make it easier to review
Possibly more for code because sometimes names are too long to type without breaks.

Reply:
Good point. I've added a note about line length. Thanks!

Reply:
Nice example!